sdk/src/Services/DynamoDBv2/Custom/DataModel/ContextInternal.cs

```diff
@@ -136,7 +136,15 @@ private Table GetTargetTableInternal<T>(DynamoDBOperationConfig operationConfig)
                 throw new ArgumentNullException("flatConfig");
 
             string tableName = GetTableName(storageConfig.TableName, flatConfig);
-            var unconfiguredTable = GetUnconfiguredTable(tableName);
+            var unconfiguredTable = GetUnconfiguredTable(tableName, flatConfig.SkipDescribeTableLoad);
+
+            if (unconfiguredTable == null && flatConfig.SkipDescribeTableLoad)
+            {
+                // In the case where we want to avoid the DescribeTable call, but failed to find a cached Table
+                // check the original table name in case a prefix has been applied or overriden
+                // This is making the assumption that storageConfig.TableName has the same structure as tableName
+                unconfiguredTable = GetUnconfiguredTable(storageConfig.TableName, flatConfig.SkipDescribeTableLoad);
```

Comment:
I think I wrote this when prototyping, but I'm still struggling to understand why. 
1. What user code will hit this code path?
2. Commenting out didn't cause any tests to fail, so could you add a test guarding that this fallback logic remains in place?

Reply:
I removed this branching since this code path will never be hit. This is because we are always caching the table info using the actual table name irrespective of whether `SkipDescribeTableLoad` is enabled or disabled.